swift/apple/Firezone/Application/FirezoneApp.swift

```diff
@@ -86,6 +91,49 @@ struct FirezoneApp: App {
       maybeShowOutdatedAlert()
     }
 
+    private func enforceSingleInstance() {
+      // Get the actual bundle identifier from the running app
+      guard let bundleId = Bundle.main.bundleIdentifier else { return }
+
+      let runningApps = NSRunningApplication.runningApplications(
+        withBundleIdentifier: bundleId
+      )
+
+      guard runningApps.count > 1 else { return }
+
+      for app in runningApps where app != NSRunningApplication.current {
+        // Try to terminate the other instance
+        var terminated = app.terminate()
```

Comment:
Is that really the behaviour that most users will want? Will that also terminate the NE?

Reply:
I'd suggest we don't do any of this termination stuff and instead just show a dialog if we detect that another instance is already running.